static/css/_forms.scss

```diff
@@ -114,6 +114,7 @@ input[type="submit"] {
 //  Writing page
 form.new-post {
   max-width: 60em;
+  overflow-y: scroll;
```

Comment:
This line should be removed, otherwise there is a weird and useless scrollbar on the right of the whole form:

![image](https://user-images.githubusercontent.com/16254623/58764689-57552d80-8562-11e9-8db3-0e7e41e44198.png)